katharsis-core/src/main/java/io/katharsis/core/properties/KatharsisProperties.java

```diff
@@ -62,4 +62,15 @@
      * @since 2.8.2
      */
     public static final String INCLUDE_AUTOMATICALLY_OVERWRITE = "katharsis.config.include.automatically.overwrite";
+
+    /**
+     * <p>
+     *     Set a boolean whether Katharsis should return a JSON API response with data field set to {@code null}
+     *     if an action returns a {@code null} value.
+     * </p>
+     * <p>
+     *     JAX-RS default default is to return a 204 response.
+     * </p>
+     */
+    public static final String NULL_DATA_RESPONSE_ENABLED = "katharsis.config.null.data.response.enabled";
```

Comment:
I am not sure how it's done, but intellij somehow is able to discover relevant properties and documentation that goes along with them. If you could figure out how that's done then we wouldn't have a magic property.

If that can't be done, then putting this property in our example code with a comment and in the default "safe" mode would at least expose it's purpose and existence